editor/editor_about.cpp

```diff
@@ -52,13 +54,25 @@ void EditorAbout::_notification(int p_what) {
 			_tpl_text->add_theme_constant_override("line_separation", 4 * EDSCALE);
 			_tpl_text->end_bulk_theme_override();
 
-			_license_text->begin_bulk_theme_override();
-			_license_text->add_theme_font_override("normal_font", font);
-			_license_text->add_theme_font_size_override("normal_font_size", font_size);
-			_license_text->add_theme_constant_override("line_separation", 4 * EDSCALE);
-			_license_text->end_bulk_theme_override();
+			license_text_label->begin_bulk_theme_override();
+			license_text_label->add_theme_font_override("normal_font", font);
+			license_text_label->add_theme_font_size_override("normal_font_size", font_size);
+			license_text_label->add_theme_constant_override("line_separation", 4 * EDSCALE);
+			license_text_label->end_bulk_theme_override();
 
 			_logo->set_texture(get_editor_theme_icon(SNAME("Logo")));
+
+			Ref<StyleBoxEmpty> empty_stylebox = memnew(StyleBoxEmpty);
+			for (ItemList *il : name_lists) {
+				il->add_theme_style_override("focus", empty_stylebox);
+				il->add_theme_style_override("selected", empty_stylebox);
+				il->add_theme_style_override("selected_focus", empty_stylebox);
```

Comment:
If you are not accessing any theme properties, but creating a new StyleBox, this can be assigned right when the list is created.

Reply:
Still not addressed. At least add `begin_bulk_theme_override()`.

Reply:
Good call. My intention was to address it. I wanted to use the same StyleBox as the an item being hovered on, but `get_theme_style()` doesn't want to work well here, so I will revert all of this workaround and put it back into the ItemList generation.